pyart/map/gates_to_grid.py

```diff
@@ -30,7 +30,6 @@ def map_gates_to_grid(
     gatefilters=False,
     map_roi=True,
     weighting_function="Barnes2",
```

Comment:
```suggestion
    weighting_function="Barnes2",
    toa=17000.0,
```

Reply:
Let's keep this in. Other than that, this looks great, we can resolve this in a separate PR